client/app/_components/ui/Calendar.tsx

```diff
@@ -0,0 +1,64 @@
+'use client'
+
+import * as React from 'react'
+import { ChevronLeft, ChevronRight } from 'lucide-react'
+import { DayPicker } from 'react-day-picker'
+import { cn } from '@/_utils'
+
+export type CalendarProps = React.ComponentProps<typeof DayPicker>
+
+function Calendar({
```

Comment:
Should the component be made with arrow components? Maybe it's better to change the name of the component since we'll make a calendar for the T1 page too. You can just say `CustomDayPicker` or something like that

Reply:
fixed it!
https://github.com/HAK2024/fairy-share/pull/80/commits/ab0cf433c152aecad248d36f906ec09045799b50